config/gnbsim.yaml

```diff
@@ -68,6 +69,7 @@ configuration:
       opc: "981d464c7c52eb6e5036234984ad0bcf"
       key: "5122250214c33e723a5dd523fc145fc0"
       sequenceNumber: "16f3b3f70fc2"
+      dnn: "internet"
```

Comment:
Can we make it list ? just a thought. We would be supporting multiple PDU sessions at some point in time. or should we keep it simple for now ?

Reply:
Can we keep it simple now? We need to map dnnList with SliceSupportList for multiple PDU sessions for each UE.
I will try to create PR for that later.

Reply:
Also, please make a note dnn and snssai is not list when it is added in `PduSessionEstablishmentRequest`.